src/form-label.scss

```diff
@@ -105,4 +105,12 @@ $block: #{$fd-namespace}-form-label;
     justify-content: space-between;
     align-items: center;
   }
+
+  &--horizontal {
+    line-height: 2.8rem;
```

Comment:
It's clear with the compact - you want compact input, use the **compact** label for it.

But when you use regular input it still needs to use the `horizontal`  label and it's not fully clear. Could you please rework it so "horizontal" will be the default? Thank you

Reply:
Hi @platon-rov , we have vertical as default. but when we want label to be aligned horizontally  with its element and top of the grid mainly in non matching no of elements we need to add this class.

hope I answered your query. If not please could you please share more details about query, that will help.